packages/zoe/test/unitTests/contracts/test-coveredCall.js

```diff
@@ -222,13 +215,21 @@ test(`zoe - coveredCall - alice's deadline expires, cancelling alice and bob`, a
     });
 
     // Bob escrows
-    const { seat: bobSeat, payout: bobPayoutP } = await zoe.redeem(
+    const { payout: bobPayoutP, outcome: bobOutcomeP } = await zoe.offer(
       bobExclOption,
       bobProposal,
       bobPayments,
     );
 
-    t.throws(() => bobSeat.exercise(), /The covered call option is expired/);
+    // TODO(msm): what's the idiom way to do the equivalent of
```

Comment:
we should use `t.rejects` https://github.com/jprichardson/tape-promise#trejectspromise-expected-msg

Reply:
Done